extensions/terminal-suggest/src/terminalSuggestMain.ts

```diff
@@ -26,25 +26,9 @@ import { createCompletionItem } from './helpers/completionItem';
 import { getFigSuggestions } from './fig/figInterface';
 import { executeCommand, executeCommandTimeout, IFigExecuteExternals } from './fig/execute';
 
-// TODO: remove once API is finalized
-export const enum TerminalShellType {
-	Sh = 1,
-	Bash = 2,
-	Fish = 3,
-	Csh = 4,
-	Ksh = 5,
-	Zsh = 6,
-	CommandPrompt = 7,
-	GitBash = 8,
-	PowerShell = 9,
-	Python = 10,
-	Julia = 11,
-	NuShell = 12,
-	Node = 13
-}
```

Comment:
Let's keep this but change the values to strings and remove the entries we don't use. That way we can prevent accidental typos in the strings as there's one source of truth.

Reply:
Good idea! would it be like: [e4a65c8](https://github.com/microsoft/vscode/pull/243274/commits/e4a65c8c5354b036d937e7bce767b606a46f0e4b)